postgres-appliance/build_scripts/post_build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd
+
+chown -R postgres:postgres "$PGHOME" "$RW_DIR"
+
+rm -fr /var/spool/cron /var/tmp
+mkdir -p /var/spool
+ln -s "$RW_DIR/cron" /var/spool/cron
+ln -s "$RW_DIR/tmp" /var/tmp
+
+for d in /etc/service/*; do
+    chmod 755 "$d/*"
```

Comment:
Name globbing is important
```suggestion
    chmod 755 "$d"/*
```